src/actions/AccountReferralActions.ts

```diff
@@ -202,15 +202,10 @@ export interface ValidateFuncs {
 
 export const getCountryCodeByIp = async (): Promise<string> => {
   const apiKey = ENV.IP_API_KEY ?? ''
-  let out = '--'
-  try {
-    const reply = await fetch(`https://pro.ip-api.com/json/?key=${apiKey}`)
-    const { countryCode } = asIpApi(await reply.json())
-    out = countryCode
-  } catch (e: any) {
-    console.error(e.message)
-  }
-  return out
+
+  const reply = await fetch(`https://pro.ip-api.com/json/?key=${apiKey}`)
```

Comment:
We don't want this routine to throw. That could cause a red error to show. Add back the try/catch and return undefined if anything throws

Reply:
Callers are already catching but yea that makes more sense